### PR TITLE
[MOD-2205] Android: Support Blob image, bump to 2.3.8

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/MOD-2205

### CODE REVIEW

**Comment by hansemannn** [comment]
@ashcoding: I have no idea how we package on the Android-side. If the PR works, can you package it? :-)

**Comment by ashcoding** [comment]
Test code app.js:-
```javascript
var MapModule = require('ti.map');
 
var win = Ti.UI.createWindow({
		fullscreen : false
	});
	
var imageOne = Ti.UI.createImageView({
  image:'/KS_nav_ui.png'
});
var blobby = imageOne.toBlob(); 

var anno = MapModule.createAnnotation({
	latitude : -33.87365,
	image : blobby,
	longitude : 151.20689,
	title : "Drag Me",
	subtitle : "Sydney is quite chill",
	draggable : true
});
 
var map = MapModule.createView({
	userLocation : true,
	mapType : MapModule.NORMAL_TYPE,
	animate : true,
	annotations : [anno],
	region : {
		latitude : -33.87365,
		longitude : 151.20689,
		latitudeDelta : 0.1,
		longitudeDelta : 0.1
	}, //Sydney
	// top : '30%'
});
 
win.add(map);
win.open();
```

File: android/src/ti/map/AnnotationProxy.java [hunk 1]
```diff
@@ -228,6 +228,17 @@ private void handleImage(Object image)
 				return;
 			}
 		}
+        
+        // Image blob
+        if (image instanceof TiBlob) {
```
Comment (ashcoding) [inline android/src/ti/map/AnnotationProxy.java]: Indentation of this section of code is off. Could you indent it?

**Comment by ashcoding** [comment]
Code reviewed and functionally tested. Working correctly. Just needs indentation on the code.

**Comment by ashcoding** [comment]
And https://github.com/appcelerator-modules/ti.map/blob/master/android/documentation/changelog.md needs updating.

**Comment by hansemannn** [comment]
Comments addressed!

**Comment by ashcoding** [comment]
All good. **Approved**

**Pull request merged**